lib/bank.rb

```diff
@@ -0,0 +1,74 @@
+$balance = 0
+
+def check_balance
+    puts "Your bank balance is $#{$balance}."
+    return $balance
+end
+
+def deposit
+    puts "How much would you like to deposit?"
+    input = gets.chomp
+    if !input.empty?
+        $balance += input.to_i
+        check_balance
```

Comment:
If this is the case, the once you deposit the bank flow stops. Shouldn't it be back to the prompt instead?